_posts/2022-05-17-cinco-de-trino-recap.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title:  "Cinco de Trino Recap: Learn how to build a efficient data lake"
+author: Brian Olsen, Brian Zhan
+excerpt_separator: <!--more-->
+---
+
+When Trino (formerly PrestoSQL) arrived on the scene almost 10 years ago, it
+immediately became known for quick analysis of big data in a data warehouse. The
```

Comment:
> of big data in a data warehouse

Has Trino ever been positioned as a data warehouse solution? Maybe rephrase?

Reply:
Trino was a replacement for Apache Hive, which is positioned as a data warehouse solution. I could spell that out so that people know what I'm talking about.

https://hive.apache.org/

Reply:
It sounds a little strange to me. I wonder if we can drop the `in a data warehouse` and simply leave it as `for quick analysis of big data.`. It somehow reads as "Trino was built for analysis of data stored in a data warehouse (e.g.: Teradata)", but maybe it's just me.